test/models.Tier.test.js

```diff
@@ -1,30 +0,0 @@
-import { expect } from 'chai';
```

Comment:
There were two files to test the `Tier` model. I merged this one to https://github.com/opencollective/opencollective-api/pull/3027/files#diff-4905f661932600450fb4d9479d7d6696